cmake/macos.cmake

```diff
@@ -150,7 +150,12 @@ if(BUILD_PACKAGE)
       "${lokinet_installer}"
       COMMAND ./seticon lokinet.icns "${lokinet_installer}.dmg"
   )
-  add_dependancies(package "${lokinet_installer}.dmg")
+  add_custom_target(apple ALL COMMAND "true" DEPENDS "${lokinet_installer}.dmg")
```

Comment:
Please don't.  The `package` target is reserved only when cpack is in use, which we do not use on Apple.  Using the `package` target gives us a target that works the same way whether using cpack (non-apple) or not (apple).

Reply:
deoending on the package target was breaking ci on mac